tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 minversion = 3.23.0
 # py is before lint because it can alter tracked files
-envlist = py,lint
```

Comment:
that is not needed. you can call py310 even without mentioning it in default envlist. That makes is easier for people as it will not force them to install multiple versions of python

Reply:
I changed it as suggested.